demo/body3d_two_stage_video_demo.py

```diff
@@ -145,6 +153,36 @@ def main():
         default='configs/_base_/filters/one_euro.py',
         help='Config file of the filter to smooth the pose estimation '
         'results. See also --smooth.')
+    parser.add_argument(
+        '--return-heatmap',
+        action='store_true',
+        default=False,
+        help='whether to return heatmap')
+    parser.add_argument(
+        '--output-layer-names',
+        nargs='*',
+        type=str,
+        help='return the output of some desired layers, '
+        'e.g. use ("backbone", ) to return backbone feature')
```

Comment:
These arguments should not be exposed to users.

Reply:
OK, should we hide these arguments in all demo scripts?